lib/formatters/csv.js

```diff
@@ -38,7 +38,38 @@ function formatCsv (sites, options = {}) {
     });
   });
 
-  return toCsvString(rows);
+  const sortedRows = rows.sort((a, b) => {
+    if (a[0] === "Index"){
+      return -1;
+    } else {
+      const a_text_hash = a[15].toLowerCase();
+      const b_text_hash = b[15].toLowerCase();
```

Comment:
This [column can be `null`](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper/blob/master/lib/formatters/csv.js#L81), so you need to account for that (I had an exception here when I ran this code).